### ISSUE TITLE
Disable cursor

### ISSUE THREAD

**Issue by 0003c9fe** [prikhi/lightdm-mini-greeter#51]
Would it be possible to add an option to disable/hide the cursor? Seeing as how all that's present is an already focused textbox, it doesn't seem necessary to have it.

**Comment by necauqua** [comment]
+1 on this, tried to do something with greeter-setup-scrips/session-setup-script but with no luck :(

And after I've set all colors to black and disabled the label the only thing left is the cursor

**Comment by NPastorale** [comment]
+1 this would be a really nice aesthetic touch to have

**Comment by prikhi** [comment]
What Linux distro are you all using? I didn't do this because on my Arch Linux setup, there is already no cursor.

**Comment by 0003c9fe** [comment]
I'm also using Arch Linux and there is a cursor.

**Comment by prikhi** [comment]
@0003c9fe I just pushed a possible fix to a new branch: 
https://github.com/prikhi/lightdm-mini-greeter/tree/gh-51-disable-cursor

Can you try the `lightdm-mini-greeter-git` package and see if it fixes the issue? You'll need to change the source array to:
```sh
source=("${pkgname}::git+https://github.com/prikhi/lightdm-mini-greeter.git#branch=gh-51-disable-cursor"
        "lightdm-mini-greeter.install")
```

**Comment by necauqua** [comment]
Arch Linux + lightdm(duh) + awesomewm the whole time

Also your branch is gone (well possibly because u've already tested it and removed it)

**Comment by prikhi** [comment]
> Also your branch is gone (well possibly because u've already tested it and removed it)

Oops, sorry - should be up again:
https://github.com/prikhi/lightdm-mini-greeter/tree/gh-51-disable-cursor

**Comment by prikhi** [comment]
I was able to reproduce this with a default Manjaro install. That fix works but it doesn't seem possible to hide the mousecursor when it's over the text input so I gotta figure out the best way to move it to some arbitrary position and then I'll merge the fix.

**Comment by prikhi** [comment]
I updated the branch to hide the cursor when it is over the background windows & main window. It also moves the cursor to the upper-left corner of the primary monitor when started.

Would appreciate someone experiencing this issue to test out he `gh-51-disable-cursor` branch before merging - I've only been testing it in a VM.

**Comment by prikhi** [comment]
Actually, was able to repro the issue w/ lightdm test-mode and it was fixed in the branch. So merged & fixed by 46d5225f3addd45af74e0174990f88f88e2e4089.

**Comment by necauqua** [comment]
So I've installed the patched version, but then never rebooted and forgot about it at all :|
Nice to have it fixed, thanks
